Fix markdownlint / spelling  issues #3

stefanroelofs opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Fix several issues related to markdownlint and spelling.


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

stefanroelofs committed:
Fix markdownlint / spelling  issues

frenck commented on source/_integrations/buienradar.markdown:
```diff
@@ -63,23 +63,23 @@ The following entities will be created:
 - **Rain chance n days ahead**: The forecasted chance for rain n days ahead (%).
 - **Sun chance n days ahead**: The forecasted chance for sun n days ahead (%).
-- **Rain n days ahead**: "The forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimeter) n days ahead; the average of minrain_1d and maxrain_1d."
+- **Rain n days ahead**: "The forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimeter) n days ahead; the average of minimum_rain_1d and maximum_rain_1d."
```
You can't just spell correct this. These are actual keys.
Do you have a PR that adjusts this in our codebase?

stefanroelofs replied:
In my configuration the entity ids are acually like this:
![image](https://user-images.githubusercontent.com/37924749/138719612-2593507a-e042-4995-b2af-9a0368ae4346.png)
These are the default entity ids, so I assumed the documentation was wrong in this case.

frenck replied:
Right, that is the key that is used internally nowadays I guess.

Maybe we should instead just refer to the human-readable name as is used on this page already.

The problem with entity IDs is, is that they aren't guaranteed either.

stefanroelofs replied:
Done 👍

frenck commented on source/_integrations/buienradar.markdown:
```diff
@@ -63,23 +63,23 @@ The following entities will be created:
 - **Minimum rain n days ahead**: "The minimum forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimeter) n days ahead."
 - **Maximum rain n days ahead**: "The maximum forecasted amount of rain in [mm](https://en.wikipedia.org/wiki/Millimeter) n days ahead."
-- **Wind azimuth n days ahead**: Where the wind is coming from in degrees, with true north at 0° and progressing clockwise for n days ahead. (derived from winddirection_1d)
+- **Wind azimuth n days ahead**: Where the wind is coming from in degrees, with true north at 0° and progressing clockwise for n days ahead. (derived from wind_direction_1d)
```
same as above

stefanroelofs replied:
Actual default configuration:
![image](https://user-images.githubusercontent.com/37924749/138720358-2b5c73e0-2d6f-4359-a528-bed4a75666ac.png)

frenck commented on source/_integrations/buienradar.markdown:
```diff
@@ -63,23 +63,23 @@ The following entities will be created:
-- **Wind force n days ahead**: "The expected windforce in [Bft](https://en.wikipedia.org/wiki/Beaufort_scale) n days ahead."
-- **Wind speed n days ahead**: "The expected windspeed in [m/s](https://en.wikipedia.org/wiki/M/s) n days ahead (derived from windforce_1d)."
+- **Wind force n days ahead**: "The expected wind force in [Bft](https://en.wikipedia.org/wiki/Beaufort_scale) n days ahead."
+- **Wind speed n days ahead**: "The expected wind speed in [m/s](https://en.wikipedia.org/wiki/M/s) n days ahead (derived from wind force_1d)."
```
same as above

stefanroelofs replied:
Same as above:
![image](https://user-images.githubusercontent.com/37924749/138720545-86d606cf-1d88-46a0-b070-9601970cd65a.png)

frenck commented on source/_integrations/cisco_webex_teams.markdown:
```diff
@@ -14,17 +14,19 @@ ha_platforms:
+2. Log into the web client at [teams.webex.com](https://teams.webex.com/),
+
+* select the room (or create a new room),
+* then copying the room ID from the URL.
```
This is a list of item 2. This should be indented to make it appear it is part of that.

stefanroelofs replied:
You are very right, fixed it.

frenck commented on source/_integrations/cisco_webex_teams.markdown:
```diff
@@ -14,17 +14,19 @@ ha_platforms:
+* select the room (or create a new room),
+* then copying the room ID from the URL.
+
+**Note: you must add the bot email (in the format `mybot@webex.bot`) as a participant to the room specified above.**
```
Let use actual notes as per developer documentation.

stefanroelofs replied:
Good idea, fixed it.

stefanroelofs committed:
Fix indentation of nested list

stefanroelofs committed:
Changed note into an actual note as per documentation

stefanroelofs committed:
Created a table for the different sensors.
This increases readability.
Also:
- Changed C into °C
- Updated other units
- Feel temperature is not correct English. Changed to wind chill factor.
- Do not refer to entityIds, use the human-readable name instead.
- Add a proper 'Note' in the bottom of the page.

stefanroelofs committed:
Renamed calendar.google.markdown file to (hopefully) resolve a merge conflict.

stefanroelofs committed:
Merge branch 'current' into Fix-markdownlint-issues-#3

frenck commented on source/_integrations/buienradar.markdown:
```diff
@@ -44,47 +44,49 @@ The Buienradar integration will set up separate sensor entities with more detail
-- **Detailed condition n days ahead**: Symbol and detailed expected condition n days ahead.
-- **Full condition (english) n days ahead**: Symbol and full expected condition n days ahead (in English).
-- **Full condition (dutch) n days ahead**: Symbol and full expected condition n days ahead (in Dutch).
+| Entity                                | Description |
```
Don't use tables, they are really problematic on mobile devices.
This change wasn't really needed.

stefanroelofs committed:
Removed markdown table.

stefanroelofs committed:
Merge branch 'Fix-markdownlint-issues-#3' of https://github.com/stefanroelofs/home-assistant.io into Fix-markdownlint-issues-#3

frenck reviewed (approved):
Thanks, @stefanroelofs 👍
